### DISCUSSION TITLE
Manage server rights with group rights

### DISCUSSION THREAD

**Discussion by MarkStephan89** [Ylianst/MeshCentral#6012] [Ideas]
Hi, 
I can not find a solution to manage the "Server Rights" per group. (only per user)
It is possible?

In my case I would manage the "No new device groups" . 

Thanks :)

**Comment by si458** [comment]
No as fair as i know, this would need to be a feature request, so open an enhancement request on github issues

**Reply by MarkStephan89** [reply]
thanks!
